plugins/org.wso2.developerstudio.eclipse.gmf.esb/model/esb.ecore

```diff
@@ -1434,7 +1434,7 @@
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="headersToExcludeInHash"
         eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="responseCodes" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"
-        defaultValueLiteral=".*"/>
+        defaultValueLiteral="*"/>
```

Comment:
Default value should be corrected to ".*". Shall we add it back and generate again?

Reply:
done